.eslintrc

```diff
@@ -2,116 +2,242 @@
+        "no-implied-eval": ["error"],
+        "no-iterator": ["error"],
+        "no-labels": ["error"],
+        "no-lone-blocks": ["error"],
+        "no-loop-func": ["error"],
+        "no-magic-numbers": ["off"],
+        "no-multi-spaces": ["off"],
+        "no-new": ["error"],
+        "no-new-func": ["error"],
+        "no-new-wrappers": ["error"],
+        "no-octal": ["error"],
+        "no-octal-escape": ["error"],
+        "no-param-reassign": ["error"],
+        "no-proto": ["error"],
+        "no-redeclare": ["error"],
+        "no-return-assign": ["error"],
+        "no-script-url": ["error"],
+        "no-self-assign": ["error"],
+        "no-self-compare": ["error"],
+        "no-sequences": ["error"],
+        "no-throw-literal": ["error"],
+        "no-unmodified-loop-condition": ["error"],
+        "no-unused-expressions": ["error"],
+        "no-unused-labels": ["error"],
+        "no-useless-call": ["error"],
+        "no-useless-concat": ["error"],
+        "no-void": ["error"],
+        "no-warning-comments": ["warn"],
+        "no-with": ["error"],
+        "wrap-iife": ["error"],
+        "yoda": ["error", "never" ],
 
         // strict mode
-        "strict": [ 2, "global" ],
+        "strict": ["error", "global" ],
 
         // variables
-        "no-catch-shadow": [ 2 ],
-        "no-delete-var": [ 2 ],
-        "no-shadow": [ 2 ],
-        "no-shadow-restricted-names": [ 2 ],
-        "no-undef": [ 2 ],
-        "no-undef-init": [ 2 ],
-        "no-undefined": [ 2 ],
-        "no-unused-vars": [ 2, { "vars": "all", "args": "none" } ],
-        "no-use-before-define": [ 2, "nofunc" ],
+        "no-catch-shadow": ["error"],
+        "no-delete-var": ["error"],
+        "no-shadow": ["error"],
+        "no-shadow-restricted-names": ["error"],
+        "no-undef": ["error"],
+        "no-undef-init": ["error"],
+        "no-unused-vars": ["error", { "vars": "all", "args": "none" } ],
+        "no-use-before-define": ["error", "nofunc" ],
 
         // node.js
-        "callback-return": [ 2, [ "callback", "cb", "next", "innerCb" ]],
-        "global-require": [ 2 ],
-        "handle-callback-err": [ 2, "^.*(e|E)rr" ],
-        "no-mixed-requires": [ 2 ],
-        "no-new-require": [ 2 ],
-        "no-path-concat": [ 2 ],
-        "no-process-exit": [ 0 ],
+        "callback-return": ["error", [ "callback", "cb", "cb1", "cb2", "cb3", "next", "innerCb", "done" ]],
+        "global-require": ["error"],
+        "handle-callback-err": ["error", "^.*(e|E)rr" ],
+        "no-mixed-requires": ["error"],
+        "no-new-require": ["error"],
+        "no-path-concat": ["error"],
+        "no-process-exit": ["error"],
 
         // stylistic
-        // turn on the few on that aren't handled by JSCS.
-        "consistent-this": [ 2, "self" ],
-        "no-array-constructor": [ 2 ],
-        "no-nested-ternary": [ 2 ],
-        "no-new-object": [ 2 ],
+        "array-bracket-newline": ["error", "consistent" ],
```

Comment:
Isn't styling handled by prettier now? Are these just duplicating prettier rules?

Reply:
Yes, stylistic rules should be removed